src/core/operations_problem.jl

```diff
@@ -470,6 +468,41 @@ function _build!(
     return
 end
 
+function get_variables_value(op_m::OperationsProblem)
+    results_dict = Dict{Symbol, DataFrames.DataFrame}()
+    for (k, v) in get_variables(op_m.psi_container)
+        results_dict[k] = axis_array_to_dataframe(v)
+    end
+    return results_dict
+end
+
+function get_dual_values(op_m::OperationsProblem; kwargs...)
+    cons = get(kwargs, :constraints_duals, nothing)
+    isnothing(cons) && return Dict{Symbol, Any}()
+    results_dict = Dict{Symbol, DataFrames.DataFrame}()
+    for c in cons
+        v = get_constraint(op_m.psi_container, c)
+        results_dict[c] = axis_array_to_dataframe(v)
+    end
+    return results_dict
+end
+
+function get_parameters_value(op_m::OperationsProblem)
+    # TODO: Still not obvious implementation since it needs to get the multipliers from
+    # the system
+    params_dict = Dict{Symbol, DataFrames.DataFrame}()
+    parameters = get_parameters(op_m.psi_container)
+    isnothing(parameters) && return params_dict
+    isempty(parameters) && return params_dict
+    for (k, v) in parameters
+        !isa(v.update_ref, UpdateRef{<:PSY.Component}) && continue
+        params_key_tuple = decode_symbol(k)
+        params_dict_key = Symbol(params_key_tuple[1], "_", params_key_tuple[3])
```

Comment:
Shouldn't this use the existing delimiter const?

Reply:
not in this case since it it for the printing of results